.github/workflows/build_and_test.yml

```diff
@@ -206,6 +206,7 @@ jobs:
       GITHUB_PREV_SHA: ${{ github.event.before }}
       SPARK_LOCAL_IP: localhost
       SKIP_UNIDOC: true
+      SKIP_MIMA: true
```

Comment:
Can we skip this in SparkR build too?

Reply:
We might have to skip here too: https://github.com/apache/spark/blob/9f20c62ff0c94a790bfd1fd925fd62fcf1ecc05a/.github/workflows/build_and_test.yml#L721

Reply:
+1 for the idea.